xwiki-pro-macros-api/src/main/java/com/xwiki/macros/note/internal/macro/NoteMacro.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package com.xwiki.macros.note.internal.macro;
```

Comment:
Why the ".macro" suffix. Isn't it redundant if the package is prefixed with ``com.xwiki.macros``?

Reply:
I agree that it is redundant but two other macros already had this structure, `userlist` and `userprofile`. I went along with this. If I change it here, I should also change it on the other two. WDYT is the best option? to leave it as is or change them all. I think we should go with the latter.

Reply:
It's not a problem to rename the internal packages but for public packages this means an API breakage. Let's leave it like this for now, for consistency, it's not a big deal.